pkg/controllers/vdb/install_reconciler.go

```diff
@@ -69,6 +70,16 @@ func (d *InstallReconciler) Reconcile(ctx context.Context, req *ctrl.Request) (c
 	if err := d.PFacts.Collect(ctx, d.Vdb); err != nil {
 		return ctrl.Result{}, err
 	}
+	// We generate https conf file and skip the install phase when running
+	// the vclusterOps feature flag
+	if vmeta.UseVClusterOps(d.Vdb.Annotations) {
+		if ok, podNotRunning := d.PFacts.anyPodsNotRunning(); ok {
```

Comment:
Do we need to wait for all pods to be running? The old installer did that because it needed to sync the admintools.conf to all pods. But that is no longer necessary. We may need to prevent going to the next actor until all pods have done the install. But that shouldn't prevent generating the certs for the pods that are running.